Client/src/Pages/Auth/Login.jsx

```diff
@@ -533,15 +421,12 @@ const Login = () => {
 					},
 				}}
 			>
-				{step === 0 ? (
-					<LandingPage onContinue={() => setStep(1)} />
-				) : step === 1 ? (
+				{step === 1 ? (
```

Comment:
This can be changed to 0 to maintain 0 indexing

Reply:
Fixed in : https://github.com/bluewave-labs/checkmate/commit/76bfb7717ac3eada7d94e11a4ce9291119b45081